Carmu/Sources/Views/SessionStart/SessionStartDriverView.swift

```diff
@@ -203,3 +225,69 @@ final class DriverFrontView: UIView {
         todayCrewMemeberLabel.text = "1"
     }
 }
+
+// TODO: - 실제 데이터로 변경
+extension DriverFrontView: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        print("count ", crewData?.crewStatus.count ?? 0)
+        return crewData?.crewStatus.count ?? 0
+    }
+    func collectionView(_ collectionView: UICollectionView,
+                        cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
```

Comment:
팀 코드 컨벤션 상 메서드 사이와 익스텐션 상단 사이에는 한 칸 씩 띄워주시는게 좋을 것 같습니다!
```suggestion
extension DriverFrontView: UICollectionViewDataSource {

    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
        print("count ", crewData?.crewStatus.count ?? 0)
        return crewData?.crewStatus.count ?? 0
    }
    
    func collectionView(_ collectionView: UICollectionView,
                        cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
```

Reply:
아앗 넵 !